config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
- Could you call  `loadMessages` repetitively? What does it do override existing messages, merge them?
- Having multi-levels is good, a dependent extension should be able to use messages from a depending extension.
- We always can load json files depending on locale and merge them, no need to declare everything in one place.